designer-demo/src/composable/http/mock.js

```
@@ -0,0 +1,939 @@
+  {
+    url: /\/material-center\/api\/block\/tenants/,
+    response: async () => {
+      const getAvailable = async () => {
+        const response = await fetch(`/mock/block/available/1.json`)
+        return response.json()
+      }
+
+      const blocks = await getAvailable()
+      const tenants = []
+
+      blocks &&
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)